FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderUnitTests.swift

```diff
@@ -137,7 +139,15 @@ final class ModelDownloaderUnitTests: XCTestCase {
     // This is an example of a functional test case.
     // Use XCTAssert and related functions to verify your tests produce the correct
     // results.
-    let modelDownloader = ModelDownloader()
+    guard let testApp = FirebaseApp.app() else {
+      XCTFail("Default app was not configured.")
+      return
+    }
+
+    let modelDownloader = ModelDownloader.modelDownloader()
+
+    let modelDownloaderWithApp = ModelDownloader.modelDownloader(app: testApp)
```

Comment:
Do we want to assert that these are the same? Both are using the default app.

Reply:
Great call! I added the test.